p2p/net/conn/dial.go

```diff
@@ -48,11 +53,15 @@ func (d *Dialer) Dial(ctx context.Context, raddr ma.Multiaddr, remote peer.ID) (
 
 		if d.PrivateKey == nil {
 			log.Warning("dialer %s dialing INSECURELY %s at %s!", d, remote, raddr)
+			log.Event(ctx, "connDialInsecure", logdial)
 			connOut = c
 			return
 		}
+
+		defer log.EventBegin(ctx, "connDialEncrypt", logdial).Done()
```

Comment:
This event isn't needed. Doesn't provide new information that `connDial` didn't already provide.